microsoftgraph/microsoft-graph-explorer-v2
# Adds the Feature: sidebar component

## Overview

Adds a filterable set of categorized sample queries on the sidebar component

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Load the app
* Check the left side of the app and see the sidebar component
* Click on the samples.
* Search for the particular sample you want to fire
